code/datums/preferences.dm

```diff
@@ -2436,3 +2386,17 @@ var/global/list/female_screams = list("female", "femalescream1", "femalescream2"
 /proc/crap_checkbox(var/checked)
 	if (checked) return "&#9745;"
 	else return "&#9744;"
+
+/proc/character_preview_icon(datum/appearanceHolder/AH, datum/mutantrace/MR = null, direction = SOUTH)
+	var/mob/living/carbon/human/H = new(null, AH)
```

Comment:
not a big fan of this practice since humans don't cleanly GC currently iirc, but I think this is better than current

What do you think @pali6?

Reply:
Could keep one global mannequin human.

Reply:
I'll see if I can set that up.